game_event/models.py

```diff
@@ -51,3 +50,9 @@ def validate_game_event(court, ball_game, min_number_of_players, max_number_of_p
     @staticmethod
     def is_ball_game_playable_at_court(court, ball_game):
         return CourtBallGame.is_ball_game_playable(court, ball_game)
+
+    def is_event_full(self):
+        from game_event_player.models import GameEventPlayer
```

Comment:
Can we move this import to the beginning of the file?

Reply:
No, It will cause a circular dependency